### PR TITLE
[bugfix](paimon)support native and jni to read paimon for minio/cos

### PR DESCRIPTION
**Pull request by wuwenchi**
## Proposed changes

1. Use the `cos` prefix to represent the cos properties.
2. Convert cos attributes into paimon's s3 attributes.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by wuwenchi** [comment]
run buildall

**Comment by wuwenchi** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -1510,6 +1512,10 @@ public void setEnableLeftZigZag(boolean enableLeftZigZag) {
                             + "when using rewriting based on struct info"})
     public boolean materializedViewRewriteEnableContainExternalTable = false;
 
+    // only use jni scanner for external table
+    @VariableMgr.VarAttr(name = FORCE_JNI_SCANNER)
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1516]: add description field in `VariableMgr.VarAttr`

**Comment by wuwenchi** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]

**Pull request merged**